client/app/bundles/course/assessment/question/scribing/actions/scribingQuestionActionCreators.jsx

```diff
@@ -69,7 +69,7 @@ function processFields(fields) {
     getSkillIdsFromSkills(fields.question_scribing.skill_ids);
 
   if (fields.question_scribing.attachment) {
-    parsedFields.question_scribing.file = fields.question_scribing.attachment[0];
+    parsedFields.question_scribing.file = fields.question_scribing.attachment.file;
```

Comment:
Random question: do you need to save the filename as well? It is represented as `name` under `AttachmentReferences`.

Reply:
@weiqingtoh Oh yes we do. I've noticed some wrong behaviours while testing it recently. Will edit it.

Reply:
@weiqingtoh Actually the name is saved under the `fields.question_scribing.attachment.file` object. Did I answer your question? 😅

Reply:
Oh ok 👍 sounds good.